src/veh_interact.cpp

```diff
@@ -605,6 +605,10 @@ bool veh_interact::can_install_part() {
         str = base.lift_strength();
         use_aid = max_lift >= lvl;
         use_str = g->u.can_lift( base );
+        if( get_option<bool>( "DISABLE_LIFTING" ) ) {
```

Comment:
This check should be done earlier. I.e. if this option is active, there's no need to calculate `use_aid` and `use_str` at all.

Reply:
Okay, I'll see what I can do.